src/go/rpk/pkg/cli/cmd/cluster/config/get.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package config
+
+import (
+	"fmt"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/out"
+	"gopkg.in/yaml.v3"
+)
+
+func newGetCommand(fs afero.Fs) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "get <key>",
```

Comment:
The code is mostly standardized around the `[ARG]` format. If you want to change that here it'd be great, though looking at `rg`, a followup PR is necessary to switch any stragglers.

Reply:
As well, should we support getting multiple keys at a time? Is there a use case for that?

Reply:
Message is gone now that ExactArgs is used.

I don't think doing multiple key fetches is worthwhile -- we would have to define a list-parseable output format, error handling for if one key exists but others don't, etc.  The idea of `get` is that it's for spot fetches, and `export` (or using the rest api) exists for bulk loads.